tools/checkpoint_conversion/checkpoint_conversion_utils.py

```diff
@@ -20,3 +22,12 @@ def get_md5_checksum(file_path):
         for byte_block in iter(lambda: f.read(4096), b""):
             md5_hash.update(byte_block)
     return md5_hash.hexdigest()
+
+
+def file_type_extractor(archive_file_path):
```

Comment:
Maybe `extract_files_from_archive`?

Reply:
Sure! Shall do the same! To ensure the tests pass, I shall make individual PRs for each of the Conversion checkpoints.

Reply:
And many checkpoints don't use `zip/tar` files for downloading. I'm leaving them as it is for now. if there are any changes to be made to account for the same, do let me know.